views/albums.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="album-container">
+{{#each searchedArtist}}
+<h2>{{name}}</h2>
+{{/each}}
+
```

Comment:
You are trying to use searchedArtist, but you are not passing this information in your route.

Reply:
Thanks a lot for your feedback! The artist's name has been implemented 🔢  in the albums page